idaes/core/util/model_diagnostics.py

```diff
@@ -581,9 +601,8 @@ def underdetermined_variables_and_constraints(self, n_calc=1, tol=0.1, dense=Fal
             dense: If True, use a dense svd to perform singular value analysis,
             which tends to be slower but more reliable than svds
```

Comment:
Documentation of the arguments to `underdetermined_variables_and_constraints` doesn't seem to render as expected in the online webpages. Might want to add tab indent to second line of documentation for each of the arguments `ncalc`, `tol`, and `dense`.

For example, tweak
```
            dense: If True, use a dense svd to perform singular value analysis,
            which tends to be slower but more reliable than svds
```
to
```
            dense: If True, use a dense svd to perform singular value analysis,
                which tends to be slower but more reliable than svds
```